translations/TablerBundle.en.xlf

```diff
@@ -110,6 +110,26 @@
         <source>Take me home</source>
         <target>Take me home</target>
       </trans-unit>
+      <trans-unit id="948fezF"  resname="Password Reset Email Sent">
+        <source>Password Reset Email Sent</source>
+        <target>Password Reset Email Sent</target>
+      </trans-unit>
+      <trans-unit id="5FE896d" resname="Enter your email address and your password will be reset and emailed to you">
+        <source>Enter your email address and your password will be reset and emailed to you</source>
+        <target>Enter your email address and your password will be reset and emailed to you</target>
+      </trans-unit>
+      <trans-unit id="RG984sg" resname="If an account matching your email exists, then an email was just sent that contains a link that you can use to reset your password. This link will expire in %token%.">
+        <source>If an account matching your email exists, then an email was just sent that contains a link that you can use to reset your password. This link will expire in %token%.</source>
```

Comment:
One last question: What is %token%? Minutes? Hours? Does it include a duration name?
It sounds weird in german, if there is only a number?

Reply:
Yeah that's true..

Here's the code generated for that part:
https://github.com/symfony/maker-bundle/blob/7fe9fc15480d06c8368141a1b9240d40efe5106b/src/Resources/skeleton/resetPassword/twig_check_email.tpl.php#L1-L11

It's not called `token` anymore..

Two possibilities:
1. Do we use trans from another bundle ? (They [added German recently](https://github.com/symfonycasts/reset-password-bundle/releases/tag/v1.16.0))
2. Do we use OUR translations ?

Reply:
we use as many translations from the the bundle as possible.
it is way more popular and we can expect, that they have more translations.